apiary.apib

```diff
@@ -3,245 +3,212 @@ HOST: http://lore.mit.edu/
 
 # LORE
 
-Simple API on top of MIT LORE application
+A REST API on top of MIT LORE application.
```

Comment:
I didn't want to suggest adding something and leaving you with no resources. :-)  I think this text would work, but you can modify it as you wish.  I wanted to cover what LORE is, what it's for, and where to look for more info.  That paragraph is overkill but it was a simple cut and paste from the LORE spec.